app/src/main/res/drawable/ic_pin_point.xml

```diff
@@ -0,0 +1,11 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
When you have time, remind me to show you a workaround in Figma that makes it easier to extract composed icons